azure-pipelines.yml

```diff
@@ -71,7 +71,6 @@ jobs:
         XCODE_APP: 'Xcode_9.4.1'
       xcode10:
         XCODE_APP: 'Xcode_10_beta'
-        XCODE_FLAGS_FOR_TEST: -parallel-testing-enabled NO
```

Comment:
AFAIK, `xcpretty` does not recognize xcodebuild output with `-parallel-testing-enabled YES`.
And we do not yet enable parallel testing on Yams.